src/comp/comp.py

```diff
@@ -24,48 +28,52 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [humans[i].name for i in range(len(humans)) if humans[i].name[0] == 'D']
```

Comment:
your code still looks very JavaScript-y. A more python way to write this is

`a = [h.name for h in humans if h.name[0] == 'D']`